Getting started section for multiplexed samples (#85)
AlexsLemonade/scpca-docs: docs/getting_started.md

The libraries available for download on the portal have not been separated by sample (i.e. demultiplexed), and therefore contain data from multiple samples.  

Libraries containing multiplexed samples can be initially processed using the same workflow described above including removal of [low quality cells](#quality-control), [normalization](#normalization), and [dimensionality reduction](#dimensionality-reduction). 
Demultiplexing can then be used to identify the sample that each cell is from. 
Demultiplexing has already been performed using both [`Seurat::HTODemux`](https://satijalab.org/seurat/reference/htodemux) and [`DropletUtils::hashedDrops`](https://rdrr.io/github/MarioniLab/DropletUtils/man/hashedDrops.html). 
For samples where corresponding bulk RNA-sequencing data is available, {ref}`genetic demultiplexing <processing_information:Genetic demultiplexing>` was also conducted. 
The results from demultiplexing using these methods have been summarized and are present in the `colData` of the `SingleCellExperiment` object in the `_filtered.rds` file only. 

> The summary of the results will identify which sample each cell belongs to in a given library, or if a confident call cannot be identified, will report `NA`. 

Reviewer: Do we want to specify the column name itself rather than "summary of the results", i.e., `*_sampleid`?

> The `hashedDrops_sampleid`, `HTODemux_sampleid`, and `vireo_sampleid` columns in the `colData` will identify which sample each cell belongs to in a given library, or if a confident call cannot be identified, will report `NA` for the specified demultiplexing method. 

Reviewer: I reworded this slightly for what I hope is a bit more clarity.
```suggestion
The `hashedDrops_sampleid`, `HTODemux_sampleid`, and `vireo_sampleid` columns in the `colData` report the sample called for each cell by the specified demultiplexing method.
If a confident call was not made for a cell by the demultiplexing method, the column will have a value of `NA`. 
```

> For more information on how to access the demultiplexing results, see {ref}`this description of demultiplexing results <sce_file_contents:demultiplexing results>`.

Reviewer: ```suggestion
For more information on how to access the full demultiplexing results, see {ref}`this description of demultiplexing results <sce_file_contents:demultiplexing results>`.
```

If desired, the sample calls identified from demultiplexing can be used to separate the `SingleCellExperiment` object by sample for downstream analysis. 

```r
# view the summary of the sample calls for genetic demultiplexing
# genetic demultiplexing results are stored in the vireo_sampleid column found in the colData

> table(multiplexed_sce$vireo)
> 
> # identify cells that are assigned to sample A
> sampleA_cells <- multiplexed_sce$vireo == "sampleA"

Reviewer: Fixing column name, adding `which` with explanatory comment.
```suggestion
table(multiplexed_sce$vireo_sampleid)

# identify cells that are assigned to sample A (use `which` to remove NA)
sampleA_cells <- which(multiplexed_sce$vireo_sampleid == "sampleA")
```
